cohorts/utils.py

```diff
@@ -16,6 +16,22 @@
 
 import re
 import warnings
+from collections import namedtuple
+
+class DataFrameHolder(namedtuple("DataFrameHolder", ["cols", "df"])):
+    """Holds a DataFrame along with associated columns of interest."""
+    def return_self(self, return_cols):
+        """Either return the DataFrame or the DataFrameHolder."""
+        if return_cols:
+            return self
+        else:
+            return self.df
+
+    @staticmethod
+    def return_obj(cols, df, return_cols=False):
```

Comment:
It can be simply `init`ed with `DataFrameHolder(cols, df)`; I use that in a few places; but this `return_obj` method is to further allow *either* the df or the holder object to be returned depending on the value of `return_cols`. Just a shortcut since that needs to happen a fair amount.